app/javascript/components/upload/form-components.jsx

```diff
@@ -100,8 +115,8 @@ export function SaveDeleteButtons({ file, updateFile, saveFile, deleteFile, vali
       onHide={() => setShowConfirmDeleteModal(false)}
       animation={false}>
       <Modal.Body className="">
-        Are you sure you want to delete { file.name }?<br/>
-        <span>The file will be removed from the workspace and all corresponding database records deleted.</span>
+        Are you sure you want to delete {file.name}?<br /><br />
+        <span>All corresponding database records will be deleted. The file <strong>{deleteText}</strong></span>
```

Comment:
Aligning with the simpler duplicate of this in `ExpandableFileForm.jsx`:

```suggestion
        <span>All corresponding database records will be deleted. The file will <strong>{deleteText}</strong></span>
```